packages/switch/README.md

```diff
@@ -24,41 +24,28 @@ When looking to leverage the `Switch` base class as a type and/or for extension
 import { Switch } from '@spectrum-web-components/switch';
 ```
 
-## Example
+### Anatomy
+
+A switch consists of a switch input and slotted label.
 
 ```html
-<sp-switch label="Switch" onclick="spAlert(this, '<sp-switch> clicked!')">
-    Switch
-</sp-switch>
+<sp-switch>Email notifications</sp-switch>
 ```
 
-### Standard switch buttons
-
-Standard switches are the default style for switches. They are optimal for
-application panels where all visual elements are monochrome in order to direct
-focus to the content.
-
-```html-live
-<div style="display: flex; justify-content: space-between;">
-    <div style="display: flex; flex-direction: column;">
-        <h4 class="spectrum-Heading--subtitle1">Default</h4>
-        <sp-field-group selected="first" name="example" vertical>
-            <sp-switch value="off">Switch Off</sp-switch>
-            <sp-switch value="on" checked>Switch On</sp-switch>
-        </sp-field-group>
-    </div>
-
-    <div style="display: flex; flex-direction: column;">
-        <h4 class="spectrum-Heading--subtitle1">Disabled</h4>
-        <sp-field-group selected="first" name="example" vertical>
-            <sp-switch disabled value="off">Switch Off</sp-switch>
-            <sp-switch disabled value="on" checked>Switch On</sp-switch>
-        </sp-field-group>
-    </div>
-</div>
+#### Checked
+
+A switch can be checked by setting the `checked` property/attribute.
+
+```html demo
+<sp-field-group vertical>
+    <sp-switch>Not checked</sp-switch>
+    <sp-switch checked>Checked</sp-switch>
```

Comment:
It looks like someone copied the `<sp-radio>` docs and swapped the content for switch. `checked` **_is_** part of the `<sp-switch>` API, so this section has been added.